### DISCUSSION TITLE
alcohol content

### DISCUSSION THREAD

**Discussion by Michelone75** [universam1/iSpindel#525] [Ideas]
I am using the fruit of your project and I must say it is gorgeous. I have a question for you, is it possible to have a virtul pin to use on Blynk that returns the alcohol content of the must (I think the formula for the calculation is% vol = (IG-FG) / 7.5 + 0.5)? Sorry for my bad English

**Comment by tapetersen** [comment]
I'm only slightly familiar with Blynk but since the iSpindel can only measure current gravity (What is FG in your formula above) and you would need to know the initial gravity (IG in you formula, often called OG or original gravity as well) to calculate that you would need some way to set that to the correct value in Blynk for every started fermentation. Alcohol content is based on the difference since you started fermentation and there is no way the iSpindel itself can measure or track that. 

You could probably do that either with a field that you can edit in your Blynk app/dashboard or (not sure if it's possible in Blynk) a button that just takes the current reading and saves that as your IG value and then bases the calculations on that but that is more of a Blynk question than an iSpindel one.

**Reply by thegreatgunbantoad** [reply]
Brewfather does cover this as it knows what the Gravity is when you started your brew.
